src/Components/Browser.JS/src/Boot.Server.ts

```diff
@@ -29,7 +32,8 @@ async function boot() {
   );
 
   window['Blazor'].reconnect = async () => {
-    const reconnection = await initializeConnection(circuitHandlers);
+    // Intentionally re-evaluating configureSignalR so it can change while the app is running.
+    const reconnection = await initializeConnection(window['Blazor'].callbacks && window['Blazor'].callbacks.configureSignalR, circuitHandlers);
```

Comment:
what I had in mind for this is that you could do this in the console if you're dev-ing and see a problem. It would be a little jank mode because you have to type out a function. Does this make sense or should we just evaluate once?